src/main/java/gregtech/api/metatileentity/MetaTileEntityHolder.java

```diff
@@ -9,6 +9,7 @@
 import gregtech.api.net.packets.CPacketRecoverMTE;
 import gregtech.api.util.GTLog;
 import gregtech.api.util.GTUtility;
+import gregtech.api.util.TaskScheduler;
```

Comment:
This import looks unused.

Reply:
This is from unfixing the cme issue. Will be used after #584 is merged